src/core/account/account-files.js

```diff
@@ -120,9 +120,10 @@ export async function loadAllWalletStates(
   )
 
   // Merge all that information together:
-  const legacyWalletInfos: EdgeWalletInfo[] = [].concat(
-    ...legacyLists.map(files => files.walletInfos)
-  )
+  const legacyWalletInfos: EdgeWalletInfo[] = []
+  for (const files of legacyLists) {
+    legacyWalletInfos.push(...files.walletInfos)
+  }
```

Comment:
Why not:

```ts
const legacyWalletInfos: EdgeWalletInfo[] = legacyLists.map(files => files.walletInfos)
```

Reply:
That is not the same result. Notice the `...` within the `push`, which has the effect of concatenating all these individual arrays together into one big array.

Reply:
Oh it's a `.flatMap` operation. Okay, then your solution is fine because flow doesn't think `Array.prototype.flatMap` is a thing.

Reply:
Also, we can't use flatMap in the core because old Androids.